integration-tests/kotlin/src/test/resources/projects/kotlin-compiler-args/pom.xml

```diff
@@ -62,9 +66,14 @@
                 <artifactId>quarkus-maven-plugin</artifactId>
                 <version>${quarkus.version}</version>
                 <configuration>
-                    <compilerArgs>
-                        <arg>-Xallow-result-return-type</arg>
-                    </compilerArgs>
+                    <compilersOptions>
```

Comment:
Ah, so this is a breaking change for Kotlin?

Reply:
If so, maybe we should make `compilerArgs` be merged into all the compiler options?

Reply:
ˋcompilerArgs` are merge into the Java section of ˋcompilerOptions`, this is not a breaking for kotlin as actual ˋcompilerArgs` weren't used in the ˋKotlinCompilationProvider`. This test was ok because in recent kotlin version, this flag is not necessary anymore...

Reply:
That's good news, thanks @glefloch!